Unable to install using `:CocInstall` with the GitHub url

**Describe the bug**

It used to work (and should work) to be able to do a `:CocInstall https://github.com/scalameta/coc-metals`. However, when you do this now, you are greeted with the following.

```
[coc.nvim] error: /Users/ckipp/bin/npm install exited with 1, messages:
npm WARN deprecated mkdirp@0.5.4: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to us
e Promises in 1.x.)
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your depende
ncies to the actual version of core-js@3.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
No receipt for 'com.apple.pkg.CLTools_Executables' found at '/'.
No receipt for 'com.apple.pkg.DeveloperToolsCLILeo' found at '/'.
No receipt for 'com.apple.pkg.DeveloperToolsCLI' found at '/'.
gyp: No Xcode or CLT version detected!
gyp ERR! configure error
gyp ERR! stack Error: `gyp` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onCpExit (/Users/ckipp/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:351:16)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 19.4.0
gyp ERR! command "/Users/ckipp/bin/node" "/Users/ckipp/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /private/var/folders/fq/nx_jsnyd6550xp03czx898d40000gn/T/coc-metals-4ei7YE/node_modules/fsevents
gyp ERR! node -v v12.13.1
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coc-metals@0.7.4 build: `tsc --noEmit && parcel build src/index.ts --out-dir lib/ --target node --no-source-maps`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coc-metals@0.7.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ckipp/.npm/_logs/2020-04-02T14_20_45_167Z-debug.log
ERROR: "build" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coc-metals@0.7.4 prepare: `npx npm-run-all clean build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coc-metals@0.7.4 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ckipp/.npm/_logs/2020-04-02T14_20_45_228Z-debug.log
```

**To Reproduce** Steps to reproduce the behavior:

1. `:CocInstall https://github.com/scalameta/coc-metals`

**Expected behavior**

This should correctly install the plugin

**Installation:**

- Operating system: macOS
- Vim or nvim version: Nvim latest

**Search terms**
CocInstall

So this seems to have also happened to other MacOS users that have upgraded to Catalina. Doing the following helped get rid of a handful of those errors.

```
sudo rm -rf $(xcode-select -print-path)
xcode-select --install
```
This was taken from this conversation:
https://github.com/schnerd/d3-scale-cluster/issues/7#issuecomment-550579897

However, still having issues.
```
[coc.nvim] error: /Users/ckipp/bin/npm install exited with 1, messages:
npm WARN deprecated core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your depende
ncies to the actual version of core-js@3.
npm WARN deprecated mkdirp@0.5.4: Legacy versions of mkdirp are no longer supported. Please update to mkdirp 1.x. (Note that the API surface has changed to us
e Promises in 1.x.)
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coc-metals@0.7.4 build: `tsc --noEmit && parcel build src/index.ts --out-dir lib/ --target node --no-source-maps`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coc-metals@0.7.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ckipp/.npm/_logs/2020-04-02T15_18_07_486Z-debug.log
ERROR: "build" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! coc-metals@0.7.4 prepare: `npx npm-run-all clean build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the coc-metals@0.7.4 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/ckipp/.npm/_logs/2020-04-02T15_18_07_541Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -306,11 +306,8 @@
   },
   "dependencies": {
     "coc.nvim": "0.0.77",
-    "metals-languageclient": "0.1.22",
+    "metals-languageclient": "0.1.23",
     "promisify-child-process": "3.1.4",
     "vscode-languageserver-protocol": "3.15.3"
-  },
-  "resolutions": {
-    "vscode-jsonrpc": "^5"
   }
 }

